web/src/features/charts/elements/GraphBackground.tsx

```diff
@@ -1,70 +1,67 @@
-/* eslint-disable react/jsx-handler-names */
-/* eslint-disable unicorn/no-null */
-/* eslint-disable react/display-name */
-import React from 'react';
+import { ScaleLinear, ScaleTime } from 'd3-scale';
+import React, { useCallback, useMemo } from 'react';
 
 import { detectHoveredDatapointIndex, noop } from '../graphUtils';
 
-const GraphBackground = React.memo(
-  ({
-    timeScale,
-    valueScale,
-    datetimes,
-    mouseMoveHandler,
-    mouseOutHandler,
-    isMobile,
-    svgNode,
-  }: any) => {
-    const [x1, x2] = timeScale.range();
-    const [y2, y1] = valueScale.range();
-    const width = x2 - x1;
-    const height = y2 - y1;
+interface GraphBackgroundProps {
+  timeScale: ScaleTime<number, number>;
+  valueScale: ScaleLinear<number, number>;
+  datetimes: Date[];
+  mouseMoveHandler?: (timeIndex: number | null, layerIndex: number | null) => void;
+  mouseOutHandler?: () => void;
+  isMobile: boolean;
+  svgNode: SVGSVGElement;
+  displayName?: string;
+}
 
-    // Mouse hover events
-    let mouseOutRectTimeout: string | number | NodeJS.Timeout | undefined;
-    const handleRectMouseMove = (event_: any) => {
-      if (mouseOutRectTimeout) {
-        clearTimeout(mouseOutRectTimeout);
-        mouseOutRectTimeout = undefined;
-      }
```

Comment:
I'm not sure that I understand why we had this in the first place. Do you know what the purpose of it was?

Reply:
No idea but I've learned that if something seems complicated there's probably a good reason why it's in the code, I would only remove this if I could explain why it was there!